bahati10/coa-taskforce-photo-gallery-challenge
# feature: setting up image displays without interaction changes

### What does this PR do?
This PR contains features for displaying 
images and their corresponding text
### Description of Task to be completed?
Images should be visible with all required styles 
and their corresponding text describing them
### How should this be manually tested?

1. Go to this repo [here](https://github.com/bahati10/coa-taskforce-photo-gallery-challenge)
2. Clone the repo locally to your favorite code editor
3. Open the **index.html** file with live server
4. Go to that opened tab in your browser and now you will be 
5. able to see the images and their corresponding text

### Any background context you want to provide?

#### N/A

### Screenshots (if appropriate)

#### N/A

### Questions:

#### N/A
